Modules/_datetimemodule.c

```diff
@@ -4663,7 +4663,10 @@ static PyMethodDef time_methods[] = {
     {"isoformat",   (PyCFunction)(void(*)(void))time_isoformat,        METH_VARARGS | METH_KEYWORDS,
      PyDoc_STR("Return string in ISO 8601 format, [HH[:MM[:SS[.mmm[uuu]]]]]"
                "[+HH:MM].\n\n"
-               "timespec specifies what components of the time to include.\n")},
+               "The optional argument timespec specifies the number of "
+               "additional terms of the time to include. Valid options are "
+               "'auto', 'hours', 'minutes', 'seconds', 'milliseconds' and "
+               "'microseconds'.\n")},
```

Comment:
This is one very long doc-string line, which needs to be broken up with `\n`.

(below as well)